packages/office-ui-fabric-react/src/components/Calendar/Calendar.test.tsx

```diff
@@ -99,7 +99,7 @@ describe('Calendar', () => {
 
     it('Verify day picker header', () => {
       let today = new Date();
-      let monthName = dayPickerStrings.months[today.getMonth()];
+      let monthName = dayPickerStrings.shortMonths[today.getMonth()];
```

Comment:
I thought this was more of a style change, the months need to be short or the layout will break with longer months.